.github/workflows/get-new-package-version/action.yml

```diff
@@ -12,8 +12,8 @@ runs:
     - shell: bash
       run: |
             CLEAN_PACKAGE_VERSION=$(node -p "require('./package.json').version")
-            PACKAGE_VERSION=${CLEAN_PACKAGE_VERSION}-${{ inputs.CHANNEL }}-rc
+
+            PACKAGE_VERSION=${CLEAN_PACKAGE_VERSION}-${{ inputs.CHANNEL }}${{ env.POSTFIX }}
```

Comment:
Lets set POSTFIX as a parameter instead of something implicit happening somewhere else

Reply:
POSTFIX was set as an output var